frontend/src/__tests__/cypress/cypress/e2e/customServingRuntimes/CustomServingRuntimes.cy.ts

```diff
@@ -73,38 +92,239 @@ describe('Custom serving runtimes', () => {
+        pathname: '/api/templates/',
+      },
+      mockServingRuntimeTemplateK8sResource({}),
+    ).as('duplicateTemplate');
+
+    const ServingRuntimeTemplateMock = mockServingRuntimeTemplateK8sResource({
+      name: 'serving-runtime-template-1',
+      displayName: 'Multi platform',
+      platforms: [ServingRuntimePlatform.SINGLE],
+      apiProtocol: ServingRuntimeAPIProtocol.GRPC,
+    });
+
+    cy.intercept(
+      { pathname: '/api/templates/opendatahub' },
+      mockK8sResourceList([...initialMock, ServingRuntimeTemplateMock]),
+    ).as('refreshServingRuntime');
+
     servingRuntimes.getRowById('template-1').find().findKebabAction('Duplicate').click();
     servingRuntimes.findAppTitle().should('have.text', 'Duplicate serving runtime');
+    cy.url().should('include', '/addServingRuntime');
+
+    servingRuntimes.shouldDisplayAPIProtocolValues([
+      ServingRuntimeAPIProtocol.REST,
+      ServingRuntimeAPIProtocol.GRPC,
+    ]);
+    servingRuntimes.shouldSelectAPIProtocol(ServingRuntimeAPIProtocol.GRPC);
+    servingRuntimes.findSubmitButton().should('be.enabled');
+    servingRuntimes.findSubmitButton().click();
+
+    cy.wait('@duplicateServingRuntime').then((interception) => {
+      expect(interception.request.body.metadata).to.eql({
+        name: 'template-1-copy',
+        annotations: { 'openshift.io/display-name': 'Copy of Multi Platform' },
+        labels: { 'opendatahub.io/dashboard': 'true' },
+        namespace: 'opendatahub',
+      });
+    });
+
+    cy.wait('@duplicateTemplate').then((interception) => {
+      expect(interception.request.body.metadata.annotations).to.eql({
+        'opendatahub.io/modelServingSupport': '["single"]',
+        'opendatahub.io/apiProtocol': 'gRPC',
+      });
+      expect(interception.request.body.objects[0].metadata).to.eql({
+        name: 'template-1-copy',
+        annotations: { 'openshift.io/display-name': 'Copy of Multi Platform' },
+        labels: { 'opendatahub.io/dashboard': 'true' },
+      });
+    });
+    cy.wait('@refreshServingRuntime');
+
+    servingRuntimes
+      .getRowById('serving-runtime-template-1')
+      .shouldHaveAPIProtocol(ServingRuntimeAPIProtocol.GRPC);
   });
 
   it('should edit a serving runtime', () => {
+    cy.intercept(
+      {
+        method: 'POST',
+        pathname: '/api/servingRuntimes',
+      },
+      {
+        delay: 500,
+        body: mockServingRuntimeTemplateK8sResource({}).objects,
+      },
+    ).as('editServingRuntime');
+    cy.intercept(
+      {
+        method: 'PATCH',
+        pathname: '/api/templates/opendatahub/template-1',
+      },
+      mockServingRuntimeTemplateK8sResource({}),
+    ).as('editTemplate');
+
     servingRuntimes.getRowById('template-1').find().findKebabAction('Edit').click();
-    servingRuntimes.findAppTitle().should('have.text', 'Edit Multi Platform');
+    cy.url().should('include', '/editServingRuntime/template-1');
+    cy.get('h1').should('contain', 'Edit Multi Platform');
```

Comment:
Why remove `servingRuntimes.findAppTitle().should('have.text', 'Edit Multi Platform');` and the use `cy.get` directly?

Reply:
ooh, I'm sorry. it was by mistake. I will update it.